pkg/apis/ignite/v1alpha2/types.go

```diff
@@ -202,6 +202,29 @@ type VMNetworkSpec struct {
 	Ports meta.PortMappings `json:"ports,omitempty"`
 }
 
+// VMStorageSpec defines the VM's Volumes and VolumeMounts
+type VMStorageSpec struct {
+	Volumes      []Volume      `json:"volumes,omitempty"`
+	VolumeMounts []VolumeMount `json:"volumeMounts,omitempty"`
+}
+
+// Volume defines named storage volume
+type Volume struct {
+	Name        string             `json:"name"`
```

Comment:
add validation that this must be set, and must match something in volumemounts

Reply:
Done in the validation added in ca75db1.